integration-tests/src/test/java/org/zowe/apiml/integration/authentication/oauth2/OktaOauth2Test.java

```diff
@@ -45,13 +46,15 @@ public class OktaOauth2Test {
     private static final String VALID_TOKEN_NO_MAPPING = SecurityUtils.validOktaAccessToken(false);
     private static final String EXPIRED_TOKEN = SecurityUtils.expiredOktaAccessToken();
 
+    @SuppressWarnings("unused")
     private static Stream<Arguments> validTokens() {
         return Stream.of(
             Arguments.of(VALID_TOKEN_WITH_MAPPING),
             Arguments.of(VALID_TOKEN_NO_MAPPING)
         );
     }
 
+    @SuppressWarnings("unused")
```

Comment:
is it possible to provide some comments?

Reply:
It's actually not a warning in intellij, apparently intellij does identify this method being used from a `@ParameterizedTest` method source. But on eclipse-based compilers such as in vscode it shows the method as unused. Should I add something like this as a comment?

Reply:
I think it would be good to have some explanation

Reply:
another option I was thinking is to make these package-private, it would remove the unwanted warnings, and these are methods for testing purposes only. Would that be better?

Reply:
I don't know why it's private now so I see no issue with this change